tests/dummy/app/routes/map.js

```diff
@@ -27,4 +28,19 @@ export default EditMapRoute.extend(EditFormRouteOperationsIndicationMixin, {
     this._super(...arguments);
   },
 
+  actions: {
+    willTransition(transition) {
+        if (!this.controller.get("showSpinner")) {
+        this.controller.set('showSpinner', true);
```

Comment:
Можно вынести из условия и сменить один раз
```suggestion
        this.controller.toggleProperty('showSpinner');
```